x-pack/plugins/enterprise_search/public/applications/app_search/components/source_engines/source_engines_logic.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { kea, MakeLogicType } from 'kea';
+
+import { Meta } from '../../../../../common/types';
+
+import { flashAPIErrors } from '../../../shared/flash_messages';
+import { HttpLogic } from '../../../shared/http';
+import { EngineLogic } from '../engine';
+import { EngineDetails } from '../engine/types';
+
+interface EnginesAPIResponse {
+  results: EngineDetails[];
+  meta: Meta;
+}
+
+interface SourceEnginesLogicValues {
+  dataLoading: boolean;
+  sourceEngines: EngineDetails[];
+}
+
+interface SourceEngineLogicActions {
```

Comment:
naming consistency

```suggestion
interface SourceEnginesLogicActions {
```

Reply:
I addressed this in [cc65416](https://github.com/elastic/kibana/pull/96896/commits/cc654166ea55a01918746e7978589df9ca02a644)